### DISCUSSION TITLE
Ability to change font back to "Inter".

### DISCUSSION THREAD

**Discussion by electricbolt** [GitbookIO/community#418] [Feature Requests]
I'd like the ability to change the font back to the "Inter" font, from the all new "Favorit" font, which is ... not my favourite. Could you please allow the font to be customized on the Plus plan, to select the original "Inter" font or the new "Favorit" font. The other fonts on the customization page can continue to be only selectable using the Pro plan.
Thanks.

**Comment by aikoii** [comment]
@electricbolt Thanks for your feedback! You can now choose Inter for your published content 🙂, and we're working on a solution for the Editor 🤞

**Comment by electricbolt** [comment]
Thanks, that's magic.
